src/record/dynamic/value/dict.rs

```diff
@@ -0,0 +1,134 @@
+use fusio::Write;
+use fusio_log::{Decode, Encode};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[repr(u8)]
+pub enum DictionaryKeyType {
+    Int8,
+    Int16,
+    Int32,
+    Int64,
+    UInt8,
+    UInt16,
+    UInt32,
+    UInt64,
+}
```

Comment:
```suggestion
#[repr(u8)]
pub enum DictionaryKeyType {
    Int8 = 0,
    Int16,
    Int32,
    Int64,
    UInt8,
    UInt16,
    UInt32,
    UInt64,
}
```

and using `DictionaryKeyType::Int16 as u8` as below